daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -400,6 +400,7 @@ func (l *logStream) Name() string {
 	return name
 }
 
+// BufSize returns the maximum bytes CloudWatch can handle.
```

Comment:
Interesting that the linter didn't fail before this PR 🤔

Reply:
The type isn't exported.

Reply:
Ah!